components/gateway/internal/gqlschema/examples/01_get_apps.graphql

```diff
@@ -1,6 +1,6 @@
 query {
```

Comment:
dear reviewer: please ignore examples for the first time round, I will update it when we agree on the schema.